packages/ember-data/tests/integration/store_test.js

```diff
@@ -171,7 +171,6 @@ test("destroying the store correctly cleans everything up", function() {
   Ember.run(store, 'destroy');
 
   equal(car.get('person'), null, "expected car.person to no longer be present");
-  equal(person.get('cars'), undefined, "expected person.cars to be empty");
```

Comment:
Computed properties are no longer removed when an object is destroy now that they are not stored on the meta object.